docs/webhooks/index.html

```diff
@@ -2,7 +2,8 @@
 <html lang="en">
 
 <head>
-    <link rel="icon" type="image/png" href="https://codesignal.s3.amazonaws.com/uploads/1553569752846/favicon.png"/>
+    <link rel="icon" type="image/png" href="https://codesignal.com/wp-content/uploads/2020/11/CodeSignal-Logo_Circle_Blue.svg" sizes="32x32"/>
```

Comment:
Not a big deal of course, but I noticed that we're using a square icon for the product site, whereas this one is a circle.

Product favicon:
https://app.codesignal.com/img/favicon-32x32.png

Is this an intentional choice by the marketing team and/or Balraj? If so, for visual & brand consistency, should we update the product's favicon to be circular as well? If not, maybe we could use the same favicon as the product site? (Just rooting for consistency here.)

Reply:
Not an intentional choice! I just didn't notice the square favicon for product. Updating!